pkg/errors/errorMessage.go

```diff
@@ -241,10 +241,14 @@ var (
 		" manually delete the file and authenticate against the server again with the 'glasactl auth login' command. If the problem persists, contact your Galasa system administrator. Detailed cause of this problem: '%s'", 1147, STACK_TRACE_NOT_WANTED)
 	GALASA_JWT_HAS_NO_EXPIRATION_DATETIME = NewMessageType("GAL1148E: The cache of access tokens contains a java web token (jwt) in file '%s', from which an expiration time could not be extracted."+
 		" This could indicate a problem with the authentication configuration on the Galasa server. Contect your Galasa system administrator. Detailed problem : '%s'\n", 1148, STACK_TRACE_NOT_WANTED)
-	GALASA_JWT_ENCRYPTION_INVALID_GALASA_TOKEN   = NewMessageType("GAL1149E: Programming logic error: Cannot encrypt because the length of the key is too small.", 1149, STACK_TRACE_NOT_WANTED)
-	GALASA_JWT_DECRYPTION_FAILED_BASE64          = NewMessageType("GAL1150E: Programming logic error: Decryption of cached bearer token failed. Reason: %s", 1150, STACK_TRACE_NOT_WANTED)
-	GALASA_JWT_DECRYPTION_FAILED_BLOCK_TOO_SMALL = NewMessageType("GAL1151E: Programming logic error: Decryption of cached bearer token failed. Cipher is not long enough. Cipher size: %d, AES block size: %d", 1151, STACK_TRACE_NOT_WANTED)
-	GALASA_ENCRYPTION_DATA_TOO_LONG              = NewMessageType("GAL1152E: Programming logic error: Too much data passed to the encryption process. Please contact your Galasa systems administrator.", 1152, STACK_TRACE_NOT_WANTED)
+	GALASA_JWT_ENCRYPTION_INVALID_GALASA_TOKEN    = NewMessageType("GAL1149E: Programming logic error: Cannot encrypt because the length of the key is too small.", 1149, STACK_TRACE_NOT_WANTED)
+	GALASA_JWT_DECRYPTION_FAILED_BASE64           = NewMessageType("GAL1150E: Programming logic error: Decryption of cached bearer token failed. Reason: %s", 1150, STACK_TRACE_NOT_WANTED)
+	GALASA_JWT_DECRYPTION_FAILED_BLOCK_TOO_SMALL  = NewMessageType("GAL1151E: Programming logic error: Decryption of cached bearer token failed. Cipher is not long enough. Cipher size: %d, AES block size: %d", 1151, STACK_TRACE_NOT_WANTED)
+	GALASA_ENCRYPTION_DATA_TOO_LONG               = NewMessageType("GAL1152E: Programming logic error: Too much data passed to the encryption process. Please contact your Galasa systems administrator.", 1152, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_DELETE_TOKEN_FAILED              = NewMessageType("GAL1153E: Failed to delete the token with id '%v'. Reason: '%s'.", 1153, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_DELETE_TOKEN_RESPONSE_PARSING    = NewMessageType("GAL1154E: The auth tokens delete operation failed. Unable to process the error information returned from the server. Reason: '%s'", 1154, STACK_TRACE_WANTED)
+	GALASA_ERROR_FAILED_TO_COMPILE_TOKEN_ID_REGEX = NewMessageType("GAL1155E: Unable to compile the regex pattern for the provided token id. Reason: '%s'", 1155, STACK_TRACE_NOT_WANTED)
```

Comment:
I think this would also be a programming logic error since the user won't be able to do anything about it